app/models/alchemy/page/page_scopes.rb

```diff
@@ -122,6 +122,10 @@ def not_public(time = Time.current)
             )
           SQL
         end
+
+        def ransackable_scopes(_auth_object)
+          [:published, :from_current_site, :searchables, :layoutpages]
+        end
```

Comment:
This commit does not belong here. Can you please rebase the branch and remove it?